source/dub/packagemanager.d

```diff
@@ -523,7 +523,7 @@ class PackageManager {
 			if (ovr.package_ != package_ || ovr.source != src)
 				continue;
 			rep.overrides = rep.overrides[0 .. i] ~ rep.overrides[i+1 .. $];
-			writeLocalPackageOverridesFile(scope_);
+			(*rep).writeOverrides();
```

Comment:
```suggestion
			rep.writeOverrides();
```
should work, no?

Reply:
Should work, but I don't like not being explicit with pointers, as it can break in non-obvious ways (e.g. when having a pointer to an array...).